dist/components/tabs/_tabs.dust

```diff
@@ -0,0 +1,20 @@
+{?tabs}
+    <div class="c-tabs" role="tab">
+        <ul id="c-tab-controls" class="c-tabs__controls" role="tablist">
```

Comment:
I don't think we've ever had a conversation about id naming conventions, since we usually avoid them like the plague. I'm open to suggestions.

I wonder if this component should have make it possible to have unique id's in case you happen to have multiple tabs on a single page? Perhaps not very likely, but... maybe?